drivers/hud_apr/app/models/hud_apr/filters/ce_apr_filter.rb

```diff
@@ -5,20 +5,6 @@
 ###
 
 module HudApr::Filters
```

Comment:
I think we should just go all the way on this and replace instances of its use with `::Filters::HudFilterBase`